docs/releases/v10.2.0.rst

```diff
@@ -0,0 +1,153 @@
+
+PostgREST 10.2.0
+================
+
+This minor version adds bug fixes and some features that provide stability to v10.0.0. These release notes include the changes added in versions `10.1.0 <https://github.com/PostgREST/postgrest/releases/tag/v10.1.0>`_, `10.1.1 <https://github.com/PostgREST/postgrest/releases/tag/v10.1.1>`_ and `10.1.2 <https://github.com/PostgREST/postgrest/releases/tag/v10.1.2>`_. You can look at the detailed changelog and download the pre-compiled binaries on the `GitHub release page <https://github.com/PostgREST/postgrest/releases/tag/v10.2.0>`_.
+
+Features
+--------
+
+Connection Lifetime
+~~~~~~~~~~~~~~~~~~~
+
+To prevent memory leaks caused by long-lived connections, PostgREST limits their lifetime in the pool through the :ref:`db-pool-max-lifetime` configuration variable.
```

Comment:
Shortening:

```suggestion
To prevent memory leaks caused by long-lived connections, PostgREST limits their lifetime in the pool through :ref:`db-pool-max-lifetime`.
```